README.md

```diff
@@ -1,30 +1,75 @@
-# Hyperledger Fabric Shim for Java chaincode
+# Hyperledger Fabric Chaincode Java
 
-This is a Java based implementation of Hyprledger Fabric chaincode shim APIs, which enables development of chaincodes using Java language.
+[![Build Status](https://dev.azure.com/Hyperledger/Fabric-Chaincode-Java/_apis/build/status/Fabric-Chaincode-Java?branchName=master)](https://dev.azure.com/Hyperledger/Fabric-Chaincode-Java/_build/latest?definitionId=39&branchName=master)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.hyperledger.fabric-chaincode-java/fabric-chaincode-shim/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.hyperledger.fabric-chaincode-java/fabric-chaincode-shim)
+[![Rocket.Chat](https://chat.hyperledger.org/images/join-chat.svg)](https://chat.hyperledger.org/channel/fabric-java-chaincode)
 
-Application developers interested in developing smart contracts (what we call chaincode) for Hyperledger Fabric should
-read the tutorial in TUTORIAL.md file and visit
-[Chaincode for developers](https://hyperledger-fabric.readthedocs.io/en/latest/chaincode4ade.html).
-
-Contributors or early adopters who need to be able to build and test recent Java chaincode shim, should reference [FAQ.md](FAQ.md) file.
+This is a Java based implementation of Hyprledger Fabric chaincode shim APIs, which enables development of smart contracts using the Java language.
 
 This project creates `fabric-chaincode-protos` and `fabric-chaincode-shim` jar
-files for developers consumption and the `hyperledger/fabric-javaenv` docker image
+files for developers' consumption and the `hyperledger/fabric-javaenv` docker image
 to run Java chaincode.
 
-## Folder structure
+## Getting Started
+
+Application developers interested in developing Java smart contracts for Hyperledger Fabric should read the [JavaDoc](https://fabric-chaincode-java.github.io/) which inludes download information, and links to documentation and samples.
+
+## Project structure
+
+### fabric-chaincode-protos
+
+Contains the protobuf definition files used by Java shim to communicate with Fabric peer.
```

Comment:
"with a fabric peer" "with THE fabric peer"? :)

Reply:
Line 25 too

Reply:
Gone with "Fabric peers" in the expectation that there'll be lots of them :)